src/Config/ECSConfig.php

```diff
@@ -25,6 +25,7 @@
 
 /**
  * @api
+ * @phpstan-type SkipRules list<string>|array<string|class-string<Sniff|FixerInterface>, string|list<string>|null>
```

Comment:
FYI: I declared it as a PHPStan type, so it can be reused in `SimpleParameterProvider`. Additionally, I've improved key/value definitions (`string` is allowed: sniffs' identifiers as keys, single paths as values).